packages/better-auth/package.json

```diff
@@ -592,12 +605,15 @@
     }
   },
   "devDependencies": {
+    "@nestjs/common": "^11.0.10",
+    "@nestjs/core": "^11.0.1",
     "@prisma/client": "^5.22.0",
     "@types/better-sqlite3": "^7.6.12",
     "@types/pg": "^8.11.10",
     "@types/prompts": "^2.4.9",
     "@types/react": "^18.3.14",
     "better-sqlite3": "^11.6.0",
+    "concurrently": "^9.1.2",
```

Comment:
concurrently was missing from dev dependencies when I tried it. If that has changed I don't mind removing it from here